src/material/snack-bar/snack-bar.md

```diff
@@ -84,10 +84,7 @@ snackbars  opened via `snackBar.open`.
 
 If no annotations are used, all the content will be treated as text content.
 
-<!-- example({
-  "example": "snack-bar-annotated-component-example",
-  "file": "snack-bar-annotated-component-example-snack.html"
-}) -->
+<!-- example(snack-bar-annotated-component) -->
```

Comment:
should I create another PR for v15.2.x docs?

Reply:
I don't think we need to. We don't backport docs changes to older versions.